tests/c_api_test/test_.py

```diff
@@ -29,7 +29,8 @@ def find_lib_path():
     lib_path = [p for p in dll_path if os.path.exists(p) and os.path.isfile(p)]
     if not lib_path:
         dll_path = [os.path.realpath(p) for p in dll_path]
-        raise Exception('Cannot find lightgbm library file in following paths:\n' + '\n'.join(dll_path))
+        dll_path_joined = '\n\n'.join(dll_path)
+        raise Exception(f'Cannot find lightgbm library file in following paths: {dll_path_joined}')
```

Comment:
The first `\n` was to separate the first sentence from the list of paths and items inside the list were combined with only one `\n` not two.
```suggestion
        dll_path_joined = '\n'.join(dll_path)
        raise Exception(f'Cannot find lightgbm library file in following paths:\n{dll_path_joined}')
```

Reply:
Okay i will update the '/n' part and should I take back the commits that committed other files and make a fresh commit ? probably that is the way to do it. @StrikerRUS